crates/ra_ide_api/src/navigation_target.rs

```diff
@@ -193,7 +193,7 @@ impl NavigationTarget {
             buf.push_str(&format!(" {:?}", focus_range))
         }
         if let Some(container_name) = self.container_name() {
-            buf.push_str(&format!(" {:?}", container_name))
+            buf.push_str(&format!(" {}", container_name))
```

Comment:
Unrelated to PR, but I hate how one has to either to `buf.push_str(&format!())` which is ugly and is a useless allocation, or to do `write!(buf, ...).unwrap()` which forces unwrap and a trait import. Does anyone knows of a crate which has `format_to!(buf, ...)` macro, which assumes strings and does not require unwraps and trait imports?